### ISSUE TITLE
DMCA Takedown Notice

### ISSUE THREAD

**Issue by sjmcallister** [butterproject/butter-desktop#321]
I just received a DMCA takedown notice for my fork of butter desktop.  The following files were listed as infringing material:

* https://github.com/popcorn-official/popcorn-desktop/blob/master/src/app/lib/providers/tvshowtime.js
* https://github.com/popcorn-official/popcorn-desktop/blob/master/src/app/lib/providers/ysubs.js
* https://github.com/popcorn-official/popcorn-desktop/blob/master/src/app/lib/providers/tvapi.js
* https://github.com/popcorn-official/popcorn-desktop/blob/master/src/app/lib/views/torrent_collection.js
* https://github.com/popcorn-official/popcorn-desktop/blob/master/package.json

I ended up deleting my fork. Not sure what's going to happen to this.

**Comment by thekillerdev** [comment]
Got that too.
https://help.github.com/articles/remove-sensitive-data/

**Comment by xaiki** [comment]
We got those too, we believe they are a mistake and have ask for comments from @github. Still waiting on an answer from them.

**Comment by xaiki** [comment]
As a good faith move we have removed `tvapi.js` that was actually never used (plan was to release a butter-api without content infringing material but then we made butter providers)

**Comment by sjmcallister** [comment]
@xaiki That's good to hear.  Looking forward to getting this resolved.

**Comment by xaiki** [comment]
https://github.com/butterproject/butter-desktop/commit/9670072ce5564fcf2907640d7d43022dc8f3d9e3

**Comment by wjin0352** [comment]
So should I just delete that repo to be done with this notice?

**Comment by xaiki** [comment]
our fork count went from 720+ to 596 now, i guess most people are doing that, we're (still) waiting on comments from @github

**Comment by xaiki** [comment]
https://torrentfreak.com/mpaa-takedown-efforts-target-legal-popcorn-time-spin-offs-160315/

**Comment by streger** [comment]
Don't delete the forks by yourself. If you can please reply to their email stating that the updated repo no longer contains any infringing material. If more emails come in that can only be good to diversify this issue and to make a stand against bullying.

Github has to answer positively to sent email from people that forked it since there isn't a bit of infringing content anywhere in the repository and whole think from start is completely false and just a sad attempt from MPAA to disrupt completely open source project with no connection whatsoever with popcorn-time official.

All you need to do is update your repo to be in sync with current butterproject.
How-to:
https://help.github.com/articles/syncing-a-fork/
https://help.github.com/articles/pushing-to-a-remote/

Thanks for the support.

**Comment by xaiki** [comment]
FYI that's the butter project's official response to GitHub
```
Hello,

It has now been more than 24hours since your Takedown notice, we have
still to hear from our previous comment, but there are a few things we'd
like to point out.

It has been brought to our attention that you sent this notification to
all Butter forks, between them many of our clients that our now asking
us for answers, one notable entity that got notified is the Brazilian
the Ministry of Culture.

We understand that your quire is with the 'popcorn-official' fork of our
project, as you know we have no control of what our forks decide to do
with our code, we can't be liable for what they do nor can other people
forking us.

Exceding the fact that the DMCA is intended for hosted content violating
copyright (here the hosted content is the code, and we have 100%
copyright over the code), not to bully activities you don't like. We
Butter Project never released any app, nor had in our history a way to
'assist in a massive copyright infringement'.

Our project goal is to build great VOD streaming apps, this statement
'literally all of the Project's source code "could be used to locate
infringing material"—in fact, this is its very purpose.' is hence plane
wrong.

In a nutshell, these massive (and abusive ?) takedown notices are
harming us, our reputation, the trust we built with our customers, and
our community (went from 720 to 585 forks on butter-desktop).

Any quick analysis will show that there is absolutely no base for the
supposed infrigements:

 - https://github.com/butterproject/butter-desktop/blob/master/src/app/lib/providers/tvshowtime.js
   this a plugin for tvshowtime.com how is that infriging ? should we sent
   DMCA notices to all of it's implementations ?

 - https://github.com/butterproject/butter-desktop/blob/master/src/app/lib/providers/ysubs.js
   this file has never been part of our project.

 - https://github.com/butterproject/butter-desktop/blob/master/src/app/lib/providers/tvapi.js
   that file is a generic plugin for a 'tvapi' endpoint that is a
   freesoftware daemon with many plugins for different data sources, we
   removed that file as we weren't using anyway.

 - https://github.com/butterproject/butter-desktop/blob/master/src/app/lib/views/torrent_collection.js
   this feature, not enabled by default, allows you to drop a random
   torrent on butter, we'll look for that torrent on kat, but we don't
   pull content from kat, the torent is user provided.

 - https://github.com/butterproject/butter-desktop/blob/master/package.json
   how can this be infriging ? this is a standard entrypoint for any
   node-webkit app…

To make things clear:
- Butter never allowed to distribute copyright infringing content
- Butter is not affiliated in anyway with the popcorntime.sh fork, we
  don't know who they are.
- We ask GitHub to desist from being the tool used by bullies to cripple
  free software.
- We ask GitHub to back us up because this sets a stanza of what the
  DMCA is for: hosted-content copyright infrigement.

Cheers,
--
Niv Sardi, Founder @ Butter Project
```

**Comment by YIFYtorrents** [comment]
@Niv Sardi is the smartest liar the commmunity has ever seen. If you wanna set up an illegal fork, just do it and stay for it, instead of pussyfooting.

**Comment by insertc0in** [comment]
@YIFYtorrents Well, then you should ask him to lend you some of his smarts because "thinking" doesn't seem to be your strong suit.

Anyway, this whole situation is being badly handled by Github, you don't just go around distributing takedown notices to unrelated repositories, at least not based on the DMCA's that they submitted in the e-mails. Fix it, Github.

**Comment by YIFYtorrents** [comment]
@insertc0in This commit was made within the same week of popcorntime.sh relaunch. I guess it was a huge coincidence - "move update keys":
https://github.com/popcorn-official/popcorn-desktop/commit/8e6382a04fa606b4cbf60aea258aeb43f1569e19

**Comment by wallydz** [comment]
@YIFYtorrents Again you and your conspiracy theory , i wouldnt even correct you and explain you what really happen and how .sh guys fork made the update that would just teach you more so i guess i let you into your ignorance and wish you will die soon

**Comment by drkarl** [comment]
At this very moment https://github.com/popcorn-official/popcorn-desktop is being throttled for clones, I guess for the amount of concurrent people trying to clone it. It's cloning at 1Kb/s.

Also, I guess many people will upload a private "backup" fork of the project in bitbucket for what could happen...

**Comment by YIFYtorrents** [comment]
@wallydz http://popcorntime.ag/official-statement.html#Malware2

**Comment by insertc0in** [comment]
@YIFYtorrents Whatever you are trying to imply is beside the point.
The point, which you clearly don't even care about, is that Butter Project is a a legitimate project, with a legitimate purpose. The DMCAs clearly do NOT mention the Butter Project and this is why we are having this discussion, we don't care about popcorn-official - let Github handle that.
But we do care about Butter Project, we care that Github is wrongfully putting pressure on legitimate forks that have nothing, zero, to do with the DMCAs in question. As for whatever you were initially trying to imply, it's not up to me to deny the false accusations. But let me eask you, if you are having such a hard time with the competition on the wrong side of the fence, why did you come to the right side, to shed tears?

**Comment by pentateu** [comment]
I got it too.

I have synced my repo with butter-desktop and I have responded to github notification. 

Not let's see what happens next.

**Comment by xaiki** [comment]
trying to keep this thread clean from spam.

**Comment by xaiki** [comment]
so, still no reply from @github from our comments, but it seems no repository was closed down... they haven't published the DMCA request either, so we're in the dark about this issue.

we did get a contact from @github support about an abuse (on this very thread) where we were promise a 'quick response'.

**Comment by Breaking1** [comment]
Here is a great tool for anyone who don't have the time or knowledge to write a DMCA counter notice themselves. It's a simpel counter notice generator, you just check the boxes that fits your situation and it will generate a counter notice based on the information you provided.

https://lumendatabase.org/counter_notices/new

**Comment by xaiki** [comment]
thanks @Breaking1 we still think it's a mistake from @github and still are waiting to hear from them, even though their silence is somehow unsettling at this point.

**Comment by Breaking1** [comment]
Yeah, hopefully this will be resolved without any more damages. No doubt it's a mistake, i think you only got caught in the crossfire.

Maybe more people will fight GitHub and take the time to file a counter notice and stand up against this kind of bullying when they have a tool that makes it a little easier.

**Comment by alxhotel** [comment]
I think we can close this issue now.

Just for the record, DMCA takedown letter published by Github: https://github.com/github/dmca/blob/master/2016/2016-03-16-MPAA.md

**Comment by xaiki** [comment]
we still have *NOT* recieved any official response from @github, so i'd like to keep that open for the record.

**Comment by vankasteelj** [comment]
@xaiki ?

**Comment by xaiki** [comment]
this is memorabilia, @github never answered us nor anyone... when we write the history of shaddy this will be exbit A

**Comment by backus** [comment]
> this is memorabilia, @github never answered us nor anyone... when we write the history of shaddy this will be exbit A

Just wanted to say thanks @xaiki for adding this final note heh. As someone who actually does spelunk random old Github issues, historical notes are useful
